### DISCUSSION TITLE
SQL inject

### DISCUSSION THREAD

**Discussion by Datron1255** [clearpool-finance/community#11] [Bug reports]

**Comment by georgeciubotaru** [comment]
Hey @Datron1255, Can you be more specific with what is the issue here?

**Reply by georgeciubotaru** [reply]
Which website do you mean, and what data you can access?

**Reply by georgeciubotaru** [reply]
@Datron1255 Can you be more precise with what is the actual issue? If you can provide an example, that would be great.

**Reply by georgeciubotaru** [reply]
@Datron1255 We are using this explicitly this website for bug reporting. If you consider your issue a bug, please provide detailed information about that so we can classify it as a bug.

**Reply by georgeciubotaru** [reply]
Hey, @Datron1255 This information is actually public and can be accessed by anyone using multiple methods like (Etherscan, our website, and our graph API).
Just to clarify the issue is that you can access the information, or you can alter the information?

**Comment by Datron1255** [comment]


**Reply by georgeciubotaru** [reply]
If you want to upload a screenshot please wait until Github uploads your file.
